src/Hl7.Fhir.Specification/Specification/Terminology/ExternalTerminologyService.cs

```diff
@@ -25,16 +25,30 @@ public ExternalTerminologyService(BaseFhirClient client)
 
         public BaseFhirClient Endpoint { get; set; }
 
+        /// <summary>
```

Comment:
This method is an implementation of the corresponding method on ITerminologyService. Move this comment there, and replace the comment here with `<inheritdoc />`, otherwise, you'll find yourself having to repeat this comment on every implementation.